conf/debug.yaml

```diff
@@ -14,6 +14,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+http:
+  dynamic: false         # enable or disable this feature
```

Comment:
The name "dynamic" is not good enough, better to use a verb instead of an adjective as field name.

Reply:
update, use `enable`